src/Cherry/Stage/Parse/Expression/Identifier.elm

```diff
@@ -68,3 +69,10 @@ namespaceParser =
         , inner = Char.isAlphaNum
         , reserved = Set.empty
         }
+
+{-| -}
+objectFieldParser : Parser AST.Identifier
+objectFieldParser =
+    Parser.succeed AST.ObjectField
+     |. Parser.symbol "."
+     |= nameParser
```

Comment:
Can you fix the indentation on these pipes, please.
```
    Parser.succeed ...
        |. ...
        |= ...
```

Reply:
If only there were a code formatter to do this for me :upside_down_face: